src/contracts/digital-twin/digital-twin.ts

```diff
@@ -208,6 +207,14 @@ export class DigitalTwin extends Logger {
 
     const twin = new DigitalTwin(options, instanceConfig);
     await twin.ensureContract();
+
+    if (options.did) {
+      const twinIdentityId = (await twin.getDescription()).identity;
```

Comment:
can we get the identity from Line 206 (options.verifications.createIdentity(config.accountId, contractId);) is this another identity?